framework/src/main/java/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -1504,6 +1516,33 @@ private Object convert(Number number, TypeKind expectedKind) {
         }
     }
 
+    /**
+     * Converts {@code number} to {@code expectedKind} * -1
```

Comment:
This multiplication suffers a type error.  A type cannot be multiplied by -1.  I would write this more clearly.

Reply:
addressed